shell/platform/windows/windows_lifecycle_manager.cc

```diff
@@ -163,4 +183,111 @@ void WindowsLifecycleManager::BeginProcessingClose() {
   process_close_ = true;
 }
 
+// TODO(schectman): Wait until the platform channel is registered to send
+// the platform message.
+// https://github.com/flutter/engine/pull/44238
```

Comment:
Please use the tracking issue instead of a pull request :)

```suggestion
// https://github.com/flutter/flutter/issues/131616
```